cliff.toml

```diff
@@ -0,0 +1,56 @@
+# git-cliff ~ configuration file
+# https://git-cliff.org/docs/configuration
+
+[changelog]
+header = """
+# Terraform Provider ACI - Changelog\n
+All notable changes to this project will be documented in this file.\n
+"""
+# template for the changelog body
+# https://keats.github.io/tera/docs/#introduction
+# TODO find a nicer way to remove the [something] tags from the start of commits.
+body = """
+{% if version %}\
+    ## {{ version | trim_start_matches(pat="v") }} ({{ timestamp | date(format="%B %d, %Y") }})
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    {{ group | striptags | trim | upper_first }}:
+    {% for commit in commits -%}
+        - {{ commit.message | trim 
+        | trim_start_matches(pat="[bugfix]") 
+        | trim_start_matches(pat="[bug_fix]") 
+        | trim_start_matches(pat="[bugfixes]") 
+        | trim_start_matches(pat="[minor_change]") 
+        | trim_start_matches(pat="[minor_changes]") 
+        | trim_start_matches(pat="[major_change]") 
+        | trim_start_matches(pat="[major_changes]") 
+        | trim | capitalize }}
+    {% endfor -%}
+{% endfor %}\n
+"""
+footer = """
+"""
+trim = true
+postprocessors = [
+  { pattern = '.*', replace_command = 'typos --write-changes -' }
+]
+
+[git]
+conventional_commits = false
+filter_unconventional = false
+split_commits = false
+commit_preprocessors = []
+commit_parsers = [
+  { message = "^.[I|i]gnore", skip = true },
+  { message = "[D|d]eprecat", group = "<!-- 0 -->DEPRECATIONS" },
+  { message = "^.[B|b]ug", group = "<!-- 1 -->BUG FIXES" },
+  { message = "^.[M|m]inor", group = "<!-- 3 -->IMPROVEMENTS" },
+  { message = "^.[M|m]ajor", group = "<!-- 3 -->IMPROVEMENTS" },
+#  { message = ".*", group = "<!-- 5 -->OTHER" },

```

Comment:
We should list those at least when starting in a "Trivial" or "Not tagged" section as if we get external commits we might need to get them.

Reply:
Any non-tagged commits will not end up in `OTHER`. This will include external commits. 
We would just need to manually group them before release.